src/site/content/en/blog/web-platform-01-2022/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "New to the web platform in January"
+subhead: >
+  In this post discover some of the interesting features that have landed in stable and beta web browsers during January 2022. 
+description: >
+  In this post discover some of the interesting features that have landed in stable and beta web browsers during January 2022.  
+date: 2022-02-01
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/8j6GZWWy9wpuQfTOkrJy.png
+alt: Balloons launching.
+authors:
+  - rachelandrew
+tags:
+  - blog
+---
+
+## Stable browser releases
+
+In January Chrome 97, and Firefox 96 became stable.
+
+Firefox 96 ships the color function [`hwb()`](https://developer.mozilla.org/docs/Web/CSS/color_value/hwb()). This function expresses a color by hue, lightness, and blackness.
+
+{% BrowserCompat 'css.types.color.hwb' %}
+
+Also in Firefox 96 is the CSS [`color-scheme`](https://developer.mozilla.org/docs/Web/CSS/color-scheme) property. This property gives you a way to indicate the color schemes an element can be rendered in. For example, to indicate that `.widget` can be rendered in the operating system’s light or dark modes, use the following CSS.
+
+```css
+.widget {
+  color-scheme: light dark;
+}
+```
+
+With color-scheme landing in Firefox, this means the property is available in Chrome, Firefox, and Safari.
```

Comment:
```suggestion
With `color-scheme` landing in Firefox, the property is also available in Chrome, Firefox, and Safari.
```

Reply:
the point of this is that the feature is now available in all evergreen browsers, "is also" doesn't make sense here as we repeat Firefox.